src/index.js

```diff
@@ -125,6 +207,7 @@ axios.get('/api/log')
 
 const socket = io();
 socket.on('refresh table', (data) => {
```

Comment:
Hey @djcruz93 @RonLek ! We need to use `socket.emit('request_table', { startDate, endDate, ...otherData })` to ask data from server with your params instead of receiving data periodically passively. What do you guys think?

Reply:
@lolimay It can be implemented either way, at present since the crawler fetches the Github info periodically after a specified time interval in config.js, I thought of emitting it from the server itself to all the connected users at a time. Right now I am emitting it after 15 seconds from the server, if we want to implement it on the client-side will do the same on the client-side by emitting it on the client and capturing it on the server-side.

Since @RonLek is getting all the GitHub info along with the created time and implemented his code by filtering on the client-side, I thought of emitting on the server-side without getting the startDate and endDate from the client. Since the data is not large, I thought of going with the flow (filtering on the client-side). If the data had been large, then filtering on the server side only makes sense.

Reply:
ah, it makes sense.

If we're going in this way, the server should provide `created_time` for every issues/prs. Thus, we can perform filter on client.

@RonLek Is it possible in the current implementation?

Reply:
@lolimay At present, it is implemented this way only. @RonLek is getting all the created_time stamps corresponding to each PR/issue for each contributor and filtering on the client-side using the startDate and endDate variables populated by the user when one filters by date.

Reply:
@lolimay that's right. We have the `created_times` for every PR and issue with us and thus the *power* to filter client side :P

Reply:
alright, let's merge this PR to avoid blocking your work.